app/templates/components/answer-form.hbs

```diff
@@ -0,0 +1,18 @@
+<form class="mb-3">
+  <div class="panel">
+    {{#each answer.errors as |error|}}
+      <div class="alert alert-danger text-capitalize" role="alert">
+        {{error.attribute}} {{error.message}}
+      </div>
+    {{/each}}
+  </div>
+
+  <div class="form-group">
+    <label for="">Body</label>
+    {{textarea name="name" rows="8" cols="80" class="form-control" value=post.description}}
+  </div>
+
+  <div class="form-group">
+    <button type="submit" class="btn btn-primary" {{action "submitForm"}}>Post an answer</button>
```

Comment:
if you use a submit button (`type=submit`) in a `<form>`, you should handle the form's submit event.